app/(main)/(campaigns)/campaigns.tsx

```
@@ -97,11 +102,10 @@ const Campaigns = () => {
             {filteredCampaigns.length > 0 ? (
               // ? <CampaignsFilledState campaigns={filteredCampaigns} />
               <FlatList
-                data={filteredCampaigns}
+                data={[,]}
```

Biome

[error] This array contains an empty slot.

Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)